demucs/separate.py

```diff
@@ -127,13 +133,29 @@ def main():
 
         track_folder = out / track.name.rsplit(".", 1)[0]
         track_folder.mkdir(exist_ok=True)
-        for source, name in zip(sources, model.sources):
-            stem = str(track_folder / name)
+        if args.stem == None:
```

Comment:
nit: `args.stem is None` is preferred to `==`.

it would be nice to share as much possible the code, one way would be to have a `source_names` variable and `sources` and do something like :
```
if args.stem is None:
    source_names = model.sources
else:
    source_names = [args.stem, 'no_' + args.stem]
    index = ...
    sources = [sources.pop(index), sum(sources)]
for source, name in zip(sources, source_names):
    ...
```
```